CHANGE_LOG

```diff
@@ -1,7 +1,40 @@
-v0.38.0
--------
+v0.38.0 (November 24, 2021)
+---------------------------
+
+The two biggest new features for this release are the support for Python 3.10
+and the ability to cross-compile for existing LLVM targets.
```

Comment:
Maybe something like:
```suggestion
This release makes llvmlite compatible with Python 3.10. It also adds an
``abiname`` option to the target machine creation interface that mimics the same
in LLVM. Further, a large number of functions are added to the IR API to support
common uses of constant expressions. Finally, a number of bugs were fixed!
```
I think the cross compilation support is a feature of how LLVM is built only.